packages/components/organisms/f-checkout/src/mixins/logger.mixin.js

```diff
@@ -0,0 +1,18 @@
+function buildErrorLogFields (error) {
+    return {
+        ...(error && error instanceof Error && {
+            exception: error.name,
+            exceptionMessage: error.message,
+            exceptionStackTrace: error.stack
+        })
+    };
+}
+
+export default {
+    methods: {
+        logInvoker (message, data, callback, error) {
```

Comment:
Can you please add some JSDocs to these methods to explain what it does and what the parameters are?